cli.go

```diff
@@ -36,15 +37,28 @@ func (cli *CLI) Run(args []string) int {
 	cpu := runtime.NumCPU()
 	runtime.GOMAXPROCS(cpu)
 
-	for _, gp := range filepath.SplitList(os.Getenv("GOPATH")) {
```

Comment:
I'm surprised to know $GOPATH allows to have one or more paths :open_mouth: .
Let me fix them.